main.py

```diff
@@ -2,6 +2,38 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        "George Klooney",
+        "Kianu Reaves",
+        "Scarlett Keegan",
+        "Will Smith",
+        "Jaden Smith",
+        "Scarlett Johansson"
+    ]
+    actors = [tuple(actor.split()) for actor in actors]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    for actor in actors:
+        Actor.objects.create(first_name=actor[0], last_name=actor[1])
```

Comment:
You could perform unpacking during iteration without the need for indexing
```suggestion
    for first_name, last_name in actors:
        Actor.objects.create(first_name=first_name, last_name=last_name)
```